packages/studio-be/src/studio/nlu/bot/index.ts

```diff
@@ -76,6 +76,10 @@ export class Bot {
     }
   }
 
+  public async downloadModelWeights(appId: string, modelId: string) {
```

Comment:
**nitpick (non-blocking)**: if you could add a return type; i'm not sure what that NLU function returns

Reply:
Done